shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -77,6 +78,38 @@ void Reset() {
 
 #pragma mark - Private interface declaration.
 
+/**
+ * Private interface declaration for FlutterWindowDelegate.
+ *
+ * This delegate inserts the FlutterTextInputPlugin into FlutterTextFields
+ * as their field editors.
+ *
+ * The FlutterTextField are only created when accessibility services are
+ * turned on. Therefore, this delegate does nothing if none of accessibility
+ * services is on.
+ */
+@interface FlutterWindowDelegate : NSObject <NSWindowDelegate>
+
+@end
+
+/**
+ * Private interface declaration for FlutterNativeTextFieldHidder.
```

Comment:
Hider

Although really this whole line should be removed; comments should not restate obvious aspects of the language itself; anyone familiar with ObjC knows that this is an interface declaration. Just like we don't write comments like:
```
// This is declaring an integer variable:
int x;
```